### DISCUSSION TITLE
The new homepage feed is fundamentally broken

### DISCUSSION THREAD

**Discussion by aurora** [community/community#66500] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Maybe it helps, if I open another discussion rather than adding to others:

I loved the old feed: I checked it at least once a day to get updates of the people and Organisations I follow and share the same interests: to get information about new projects they either created or switched to "public" or to find new interesting repositories they starred. Something with the new feed seems to be fundamentally broken: 

I do not get any updates of the kind I got with the old feed, regardless of the filters that can be set. With the new feed, I only get unrelated/uninteresting stuff or stuff I am not so much interested in. I double-checked by comparing the new with the old feed, that is hidden, but still reachable. I tried to change the filter-settings, to no avail. I just do not get the updates I used to get before.

I fail to understand why a working feature is abandoned, by replacing it with a broken one. Thanks anyway for your work, and thanks in advance for fixing this by bringing back the old feed.

**Comment by amlcodes** [comment]
commenting here instead of opening a new issue to follow on your point.

I was sort of excited when I saw the intended changes, but almost immediately it was unusable. The feed is incredibly cluttered and unhelpful and uninteresting now. The filters need to be defined far more granularly if you want to move away from the previous feed algorithms, and that would be a tedious UX. The filter system as it is now does not seem to be working at all, I can remove one or two filters and all I can see are a couple people who followed me, even though I interact with and follow enough projects that it should be never ending with other activity. At the minimum, the filters repositories, follows, stars, and recommendations plainly do not work and will return 0 activity if selected. I don't care that github-actions or anyone else labels a PR or an issue, yet that is all that I can see, especially since pagination is completely removed. Pointed out by other users as well, information like releases are being merged without reason, the feed UI itself is cumbersome and unintuitive. This was not my experience before. I was happy to check what was going on on github, now it's immediately bothersome and removes very nearly all of the points of discovery I was relying on and expecting, including stars. The new feed is also surprisingly slow considering we got more and more accurate activity with the old feed. To point out what other people are saying, it really does feel like the people who managed, designed, and developed this project do not actually use the feature. Unfortunately, I'm honestly not sure who would use the feed as it is now. This is just begging for and makes me want to stop following people, projects, orgs, and ultimately interacting on GH.

**Comment by aurora** [comment]
![image](https://github.com/community/community/assets/304837/06083684-a714-4246-88ec-4d6941ab52a2)

These two filters, as I understand it, should show me the things that the old feed did for me. While with the new feed I've been seeing "That's all for now" since yesterday(!), the old feed shows me tons of new interesting things that the new feed didn't even show. How come?

Instead, I see - if anything - stuff of people, I do not even follow!

**Comment by dbry** [comment]
The most obvious thing missing now, no matter what I select, is when someone _I don’t know_ stars one of my repositories. This always prompts me to find out what this stranger up to, which is often interesting. Now all I see is when people _I already know_ do something. Pretty useless.

**Comment by loneicewolf** [comment]
> ### Select Topic Area
> 
> Bug
> ### Body
> 
> Maybe it helps, if I open another discussion rather than adding to others:
> 
> I loved the old feed: I checked it at least once a day to get updates of the people and Organisations I follow and share the same interests: to get information about new projects they either created or switched to "public" or to find new interesting repositories they starred. Something with the new feed seems to be fundamentally broken:
> 
> I do not get any updates of the kind I got with the old feed, regardless of the filters that can be set. With the new feed, I only get unrelated/uninteresting stuff or stuff I am not so much interested in. I double-checked by comparing the new with the old feed, that is hidden, but still reachable. I tried to change the filter-settings, to no avail. I just do not get the updates I used to get before.
> 
> I fail to understand why a working feature is abandoned, by replacing it with a broken one. Thanks anyway for your work, and thanks in advance for fixing this by bringing back the old feed.

99th' upvote from me. I agree. `:-)`

**Reply by kjvdven** [reply]
I would like to see the filter which belongs to an update (tag or something), is it a star or repository or someone I follow.

**Comment by ettaboyle** [comment]
_We've added the following update to the [Updates to your GitHub Feed](https://github.com/orgs/community/discussions/65343#top) Discussion Announcement originally posted August 29, 2003._


## _Sept. 12th Update_
_With the latest feed update on September 6, 2023, we made changes to the underlying technology of the feed to improve overall platform performance. As a result, we removed the functionality for “push events for repositories a user is subscribed to”. We don’t take these changes lightly, but as our community continues to grow tremendously, we have to prioritize our availability, user experience, and performance._ 


_Thanks to feedback from the community, we have updated our [changelog](https://github.blog/changelog/2023-09-06-updates-to-your-github-feed/) to clarify and have fixed the following bugs from the initial September 6th release:_


- _We were showing releases for organizations that you follow, instead of only showing them for repositories that you watch or star. This is now fixed._ 
- _We have also addressed a problem where 'Followed You' cards were sometimes appearing out of chronological order._
- _We do not plan to re-include “push events for repositories a user is subscribed to” for the reasons listed above, but we will continue to address feedback as it aligns with our platform goals._


_We understand that many of you are upset with the recent changes to your feed. We should have done a better job communicating recent changes and how those decisions relate to our broader platform goals. Your continued feedback is invaluable as we evolve and continue to strive to provide a first-class developer experience that helps every developer be happier and more productive._
